packages/@headlessui-react/CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Accept optional `strategy` for the `anchor` prop ([#3034](https://github.com/tailwindlabs/headlessui/pull/3034))
 - Expose `--input-width` and `--button-width` CSS variables on the `ComboboxOptions` component ([#3057](https://github.com/tailwindlabs/headlessui/pull/3057))
 - Expose the `--button-width` CSS variable on the `PopoverPanel` component ([#3058](https://github.com/tailwindlabs/headlessui/pull/3058))
+- Close the `Menu`, `Listbox`, `Popover` and `Dialog` components when the trigger disappears ([#3075](https://github.com/tailwindlabs/headlessui/pull/3075))
```

Comment:
Alphabetical 🤷‍♂️

```suggestion
- Close the `Dialog`, `Listbox`, `Menu`, and `Popover` components when the trigger disappears ([#3075](https://github.com/tailwindlabs/headlessui/pull/3075))
```

Reply:
I completely missed this one, 100% should be added there as well!